bundler/lib/bundler/cli/gem.rb

```diff
@@ -190,14 +190,23 @@ def run
 
       templates.merge!("exe/newgem.tt" => "exe/#{name}") if config[:exe]
 
-      if extension
+      if extension == "c"
         templates.merge!(
-          "ext/newgem/extconf.rb.tt" => "ext/#{name}/extconf.rb",
+          "ext/newgem/extconf-c.rb.tt" => "ext/#{name}/extconf.rb",
           "ext/newgem/newgem.h.tt" => "ext/#{name}/#{underscored_name}.h",
           "ext/newgem/newgem.c.tt" => "ext/#{name}/#{underscored_name}.c"
         )
       end
 
+      if extension == "rust"
+        templates.merge!(
+          "Cargo.toml.tt" => "Cargo.toml",
+          "ext/newgem/Cargo.toml.tt" => "ext/#{name}/Cargo.toml",
+          "ext/newgem/extconf-rust.rb.tt" => "ext/#{name}/extconf.rb",
```

Comment:
How about `extconf-rs.rb.tt` instead of `extconf-rust.rb.tt`?

Reply:
Why do you think `extconf-rs.rb.tt` is better, @hsbt?

Reply:
In case we will onboard `magnus` as proposed, it would be easier to keep it in line with `--ext` value using current `extconfig-$EXT.rb.tt` and pattern.

Reply:
> it would be easier to keep it in line with `--ext` value using current `extconfig-$EXT.rb.tt` and pattern.

👍  Thanks. We should keep it.